### DISCUSSION TITLE
Documentation for version 4

### DISCUSSION THREAD

**Discussion by ranbuch** [the-guild-org/apollo-angular#1809] [Q&A] [answered]
Hi, wondering when will the documentation for `V4` will be available or alternatively if I can use `V3` documentation instead. Also, can I use `V3` for `Angular 14`?

Thanks.

**Comment by Destreyf** [comment, marked as answer]
@ranbuch I've been using v3 documentation since the v4 launch without issues, the main reason from my understanding for the major version bump was a compatibility one due to changes required around testing with Angular 14.

If you notice anything out of place you can feel free to respond here and maybe the community here can help.
